src/plugins/orangehrmClaimPlugin/Api/ClaimEventAPI.php

```diff
@@ -79,7 +102,36 @@ public function setClaimEvent(ClaimEvent $claimEvent)
     }
 
     /**
-     * @return EndpointResult
+     * @OA\Get(
+     *     path="/api/v2/claim/events",
+     *     tags={"Claim/Claim Events"},
```

Comment:
```suggestion
     *     tags={"Claim/Events"},
```
@ManulMax what do you think?

Reply:
tags={"Claim/Events"}, is better